sqlglot/dialects/spark.py

```diff
@@ -27,6 +27,21 @@ def _str_to_date(self, expression):
     return f"TO_DATE({this}, {time_format})"
 
 
+def _unix_to_time(self, expression):
+    scale = expression.args.get("scale")
+    timestamp = self.sql(expression, "this")
+    if not scale:
+        return f"FROM_UNIXTIME({timestamp})"
+    if scale == 0:
```

Comment:
instead of assuming 0,3,9, what do you think if snowflake converts it into "seconds", "millis", "micros"

when parsing it in

Reply:
@saligrama i mean the actual string representation "seconds" so that we don't have implicit assumptions of what 0,3, and 9 mean

Reply:
It might be better to keep it as 0,3,9 for dialects that don't have a way to distinguish between unix epoch timestamp scales; that way we can do something like `new_timestamp = timestamp / (10 ** int(scale))` rather than a bunch of if statements.

Reply:
Definitely open to using that convention though - as far as my purposes go, just need to handle the Snowflake/Spark cases

Reply:
Ah ok, let me make that change

Reply:
also do exp.Literal.string("seconds")

Reply:
and if you had chosen to do numbers, exp.Literal.number(0)

Reply:
done